scripts/moosetest

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+import sys, os
+
+MOOSE_DIR = os.environ.get('MOOSE_DIR', os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+sys.path.append(os.path.join(MOOSE_DIR, 'python'))
+os.environ['MOOSE_DIR'] = MOOSE_DIR
+os.environ['PYTHONPATH'] = os.environ.get('PYTHONPATH', '') + ':' + os.path.join(MOOSE_DIR, 'python')
```

Comment:
Do these 2 need to be set here? They are set in the TestHarness right?

Reply:
fixed